README.md

```diff
@@ -33,6 +33,7 @@ _These assume you have `egghead-systems` running locally_
 - **Package management:** Yarn
 - **Scripts:** Yarn
 - **JS Flavor:** ES2016+ _(default `create-react-app`)_
+- **JS Utils:** Lodash
```

Comment:
Those folks are so smart. I think it might "just work" - I love the idea of being more reactive and definitely want to level up in Rx. I'm not 💯 on if that is a good idea or not though 😅